app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -1,16 +1,26 @@
 .gem-c-intervention {
   @include govuk-text-colour;
-  @include govuk-responsive-padding(4);
-  @include govuk-responsive-margin(4, "bottom");
+  @include govuk-responsive-padding(6, "left");
+  @include govuk-responsive-padding(6, "right");
+  @include govuk-responsive-padding(4, "top");
+  @include govuk-responsive-padding(4, "bottom");
+  @include govuk-responsive-margin(6, "bottom");
   background-color: govuk-colour("light-grey", $legacy: "grey-4");
 }
 
 .gem-c-intervention__title {
-  @include govuk-font(24, $weight: bold);
+  @include govuk-font(19, $weight: bold);
   margin-top: 0;
-  @include govuk-responsive-margin(4, "bottom");
+  @include govuk-responsive-margin(0);
+  @include govuk-responsive-padding(4, "bottom");
 }
 
 .gem-c-intervention__paragraph {
   @include govuk-font(19);
+  @include govuk-responsive-margin(0);
```

Comment:
Similar comment as above, these styles seem similar (or the same) as the standard [govuk-body](url) paragraph style, so I wonder if we can just apply that class to the relevant elements?

Reply:
I've simplified the whole markup by using `govuk-` attributes and keeping only the necessary CSS